src/ClimaAtmos/Dycore/src/VanillaAtmosDiscretizations.jl

```diff
@@ -8,12 +8,16 @@ using ..AtmosStateArrays
 
 export VanillaAtmosDiscretization
 
+
 using ...ParametersType
 using ...PlanetParameters: cp_d, cv_d, R_d, grav
-@parameter gamma_d cp_d/cv_d "Heat capcity ratio of dry air"
+using CLIMA.MoistThermodynamics
+
+@parameter gamma_d cp_d/cv_d "Heat capacity ratio of dry air"
```

Comment:
We should remove the `gamma_d` and `gdm1` here—they are exclusively dry, and we should not need them in the core.

Reply:
Noted